### PR TITLE
Update Slack plugin

### PR DESCRIPTION
**Pull request by SuneKjaergaard**
Remove the dependency on the unmaintained Slacker library and Slack access tokens which are also deprecated.

Use slack web hooks instead, which are easy to configure

### CODE REVIEW

File: app_image_generator/plugins/slack.py [hunk 1]
```diff
@@ -1,36 +1,36 @@
...
 class SlackPlugin(BasePlugin):
     def __init__(self):
-        self.slacker = Slacker(os.environ.get("SLACK_TOKEN"))
-        self.channel = os.environ.get("SLACK_CHANNEL")
+        self.webhook_url = os.environ.get("SLACK_WEBHOOK_URL")
 
     def build_succeeded(self, app, output):
-        self.slacker.chat.post_message(self.channel or '#dev', 'New %s images available:' % app, username='app-image-generator', attachments=[
-            {
+        requests.post(self.webhook_url, data({
```
Comment (guy881) [inline app_image_generator/plugins/slack.py]: `data(`?

File: app_image_generator/plugins/slack.py [hunk 2]
```diff
@@ -1,36 +1,36 @@
...
+            "attachments": [{
```
Comment (guy881) [inline app_image_generator/plugins/slack.py:11]: no channel info, did you test this? :D

File: app_image_generator/plugins/slack.py [hunk 3]
```diff
@@ -1,36 +1,36 @@
 import os
-from slacker import Slacker
 from app_image_generator.plugins.base import BasePlugin
```
Comment (guy881) [inline app_image_generator/plugins/slack.py:2]: Also, forgot to import `requests` 🚔 😄

File: app_image_generator/plugins/slack.py (hunk 2 above)
Comment (SuneKjaergaard) [reply app_image_generator/plugins/slack.py:11]: When using webhooks, you define the channel in the webhook configuration.

File: app_image_generator/plugins/slack.py (hunk 1 above)
Comment (SuneKjaergaard) [reply app_image_generator/plugins/slack.py]: Oups :D

**Comment by SuneKjaergaard** [comment]
Alright, I cleaned up the mess @guy881

**Review by guy881 — APPROVED** [review]

**Pull request merged**